did/cli.py

```diff
@@ -149,8 +149,17 @@ def parse(self):
             raise RuntimeError(
                 "Invalid date range ({0} to {1})".format(
                     opt.since, opt.until.date - delta(days=1)))
-        header = "Status report for {0} ({1} to {2}).".format(
+
+        header = "Status report for {0} ({1} to {2})".format(
             period, opt.since, opt.until.date - delta(days=1))
+        if opt.format == "md":
+            # in markdown first line must be an header
+            # using alternate syntax allowing to use did's
+            # output in commit messages as well
+            header = f"{header}\n{'=' * len(header)}"
+        else:
+            # in markdown no trailing punctuation is allowed in headings
```

Comment:
```suggestion
            # In markdown no trailing punctuation is allowed in headings
```

Just consistent comments...